### DISCUSSION TITLE
This stack is not managed by Dockge

### DISCUSSION THREAD

**Discussion by clabnet** [louislam/dockge#423] [Ask for Help] [answered]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

First install of dockge on windows 11.
I have these folders structure: 
```
-- my-stack
|---- dockge
|---- stacks
           |----- container 1
                       |------ docker-compose.yml
           |----- other containers
                       |------ docker-compose.yml
           
```
I have updated the env var 
```
    environment:
      # Tell Dockge where is your stacks directory
      #- DOCKGE_STACKS_DIR=/opt/stacks
      - DOCKGE_STACKS_DIR=../stacks                       <<===
```
but the result is always the same : **This stack is not managed by Dockge**.

### 🐻 Dockge Version

Version: 1.4.2 Frontend Version: 1.4.2

### 💻 Operating System and Arch

Windows 11

### 🌐 Browser

Google Chrome v.121.0.6167.161 (a 64 bit)

### 🐋 Docker Version

Docker version 25.0.2, build 29cf629

### 🟩 NodeJS Version

v20.7.0

**Comment by trythatagain** [comment, marked as answer]
I don't run W11, but assume docker is the same as Debian where I use OMV, can you show the following for dockge yml file.

     # Your stacks directory in the host (The paths inside container must be the same as the host)
      # ⚠️⚠️ If you did it wrong, your data could end up be written into a wrong path.
      # ✔️✔️✔️✔️ CORRECT EXAMPLE: - /my-stacks:/my-stacks (Both paths match)
      # ❌❌❌❌ WRONG EXAMPLE: - /docker:/my-stacks (Both paths do not match)
      - /srv/dev-disk-by-uuid-2b9b3914-0f0d-4737-9638-123456/compose:/opt/stacks
    environment:
      # Tell Dockge where is your stacks directory
      - DOCKGE_STACKS_DIR=/opt/stacks

I just found it easier to have my compose/stacks volume point to 'opt/stacks' and also DOCKGE_STACKS_DIR=/opt/stacks and it all works lime a charm. :)

So yours might be something like
    
     volumes:
         - my-stack/dockge/stacks:/opt/stacks
    environment:
      # Tell Dockge where is your stacks directory
        - DOCKGE_STACKS_DIR=/opt/stacks
      #- DOCKGE_STACKS_DIR=../stacks

**Reply by clabnet** [reply]
@trythatagain You are right. It work. Thank's to suggestion to volumes.
